### PR TITLE
Add FFmpeg in prerequisites list

### PR DESCRIPTION
**Pull request by Kh4L**
Signed-off-by: Serge Panev <spanev@nvidia.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -99,6 +101,8 @@ Prerequisites
    +----------------------------------------+---------------------------------------------------------------------------------------------+
    | |jpegturbo link|_ or later             | *This can be unofficially disabled. See below*                                              |
    +----------------------------------------+---------------------------------------------------------------------------------------------+
+   | |ffmpeg link|_ or later                | We recommend using version 3.4.2 and compiled following the *instructions below*.           |
```
Comment (JanuszL) [inline README.rst]:
```suggestion
   | |ffmpeg link|_ or later                | We recommend using version 3.4.2 compiled following the *instructions below*.           |
```

Comment (Kh4L) [reply README.rst]: Done

**Comment by Kh4L** [comment]
Build 591918

**Comment by klecki** [comment]
Our dockerfile.deps should use the link that's provided here.

**Comment by JanuszL** [comment]
@klecki - that is true, we could update that in this patch as well

**Pull request merged**